src/components/LoansByCategory/CategoryRowHover.vue

```diff
@@ -236,6 +242,38 @@ export default {
 			// eslint-disable-next-line no-underscore-dangle
 			return `${this.loans[this.detailedLoanIndex].__typename}:${this.loans[this.detailedLoanIndex].id}`;
 		},
+		noHoverLoan() {
+			return this.hoverLoanIndex === null;
+		},
+		hoverLoanIsLeftMost() {
+			if (this.noHoverLoan) {
+				return false;
+			}
+
+			const bodyRect = document.body.getBoundingClientRect();
+			const hoverLoanCardRect = this.$refs.hoverLoanCards[this.hoverLoanIndex].$el.getBoundingClientRect();
+			const hoverLoanCardLeft = hoverLoanCardRect.left - bodyRect.left;
+
+			const hoverCardsHolderRect = this.$refs.hoverCardsHolder.getBoundingClientRect();
+			const noArrowPaddingSide = 1;
+			const arrowWidthInRem = 2.5;
+			const pxInRem = 16;
+			const leftPadding = this.categoryRowArrowsVisible()
+				? arrowWidthInRem * pxInRem
+				: noArrowPaddingSide * pxInRem;
+			const hoverCardsHolderLeft = hoverCardsHolderRect.left - bodyRect.left + leftPadding;
+
+			const hoverCardDistanceFromLeft = hoverLoanCardLeft - hoverCardsHolderLeft;
+
+			return hoverCardDistanceFromLeft === 0;
+		},
+		hoverLoanIsRightMost() {
+			if (this.noHoverLoan) {
+				return false;
+			}
+
+			return false;
```

Comment:
I expect more to come here later...

Reply:
Hahahaha I don’t know, it looks nice so minimal ;)